src/psyclone/generator.py

```diff
@@ -21,16 +21,84 @@
 import sys
 import os
 import traceback
-from parse import parse, ParseError
-from psyGen import PSyFactory, GenerationError
-from algGen import NoInvokesError
-from config import SUPPORTEDAPIS, DEFAULTAPI, DISTRIBUTED_MEMORY
-from line_length import FortLineLength
+from psyclone.parse import parse, ParseError
+from psyclone.psyGen import PSyFactory, GenerationError
+from psyclone.algGen import NoInvokesError
+from psyclone.config import SUPPORTEDAPIS, DEFAULTAPI, DISTRIBUTED_MEMORY
+from psyclone.line_length import FortLineLength
+from psyclone.version import __VERSION__
+
+
+def handle_script(script_name, psy):
+    '''Loads and applies the specified script to the given psy layer.
+    The 'trans' function of the script is called with psy as parameter.
+    :param script_name: Name of the script to load.
+    :type script_name: string
+    :param psy: The psy layer to which the script is applied.
+    :type psy: :py:class:`psyclone.psyGen.PSy`
```

Comment:
Could you please add the exceptions as part of the function signature.

Reply:
I've added the exception details about the ones explicitly raised in the subroutine. But it also has a generic catch-all exception handling, re-raising the original exception - and I have no idea how to document this (and didn't see any example either).